gulpfile.js

```diff
@@ -0,0 +1,22 @@
+var gulp     = require('gulp'),
+    mocha    = require('gulp-mocha'),
+    babel    = require('babel'),
+    plumber  = require('gulp-plumber'),
+    istanbul = require('gulp-istanbul');
```

Comment:
I think its alright with logical grouping, which this doesn't really have yet.

So I'm not a huge fan of using an auto loader, I'm much more comfortable in this case using explicit declarations. That being said my understanding of current gulp best practices is to break tasks out into their own files. So once this grows large enough I would want to have the gulpfile just be a header file of sorts and have all the functionality logically grouped into their own files. That solves the problem of the giant require blocks and excessively long gulp files.